### DISCUSSION TITLE
第6节课作业（14班）

### DISCUSSION THREAD

**Discussion by NewDaNew** [InternLM/Tutorial#412] [作业]
最后一节课啦，冲冲

**Comment by ZZfive** [comment]
https://sincere-papyrus-971.notion.site/5035146920bd4910b17470bbd02ad9d8?pvs=4

**Comment by Nobitaxi** [comment]
https://blog.csdn.net/weixin_44288092/article/details/135747481

**Comment by Mercuryoo** [comment]
https://t3d0h0gdxm.feishu.cn/docx/T07bdVYMioVsqNxPda5czTP1nse?from=from_copylink

**Comment by xilimala** [comment]
https://zhuanlan.zhihu.com/p/679956102

**Comment by shiningwhite-cmd** [comment]
https://www.yuque.com/shanxiaobai/xt4xo5/eo9dy5ggwaqgy7i7?singleDoc# 《评测》

**Comment by lizihe11** [comment]
https://gitee.com/kiddylzh/intern-lm-tutorial-note/blob/master/task/sixth%20task.md

**Comment by SingaCycle** [comment]
https://blog.csdn.net/z1370378301/article/details/135902949  作业+笔记
